doc/flow-definition.md

```diff
@@ -1,11 +1,308 @@
-# The flow definition
+# Flow
 
-The Ruby flow file defines a `Flow` class that contains all the questions, outcomes and rules to control flow between.
+The Ruby flow file defines a subclass of `SmartAnswer::Flow` that specifies metadata, and defines all the question nodes, the outcome nodes, and the rules to control flow between the nodes.
```

Comment:
I've addressed this in a fix-up commit: e21ec46055eeab8cf9dd4720615d92a3200fedc2.